File: 09. Array/C. Modus Terbesar/c_modus_terbesar.go
package main

import (
  "fmt"
)

func main() {
  var N, ans int
  var arr [1001]int
  fmt.Scan(&N)
  for i := 0; i < N; i++ {
    var num int
    fmt.Scan(&num)
    arr[num] += 1
  }
  ans = 0
  for i := 0; i < 1001; i++ {
    if arr[i] >= arr[ans] {
      ans = i
    }
  }
  fmt.Println(ans)
}

File: 09. Array/C. Modus Terbesar/c_modus_terbesar.java
import java.util.Scanner;

class c_modus_terbesar {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int N = scan.nextInt();
        int[] arr = new int[1001];
        for (int i = 0; i < N; i++) {
            int num = scan.nextInt();
            arr[num] += 1;
        }
        scan.close();
        int ans = 0;
        for (int i = 0; i < 1001; i++) {
            if (arr[i] >= arr[ans]) {
                ans = i;
            }
        }
        System.out.println(ans);
    }
}